scout/adapter/mongo/matchmaker.py

```diff
@@ -104,14 +103,15 @@ def case_mme_update(self, case_obj, user_obj, mme_subm_obj):
 
         # create events for subjects add in MatchMaker for this case
         institute_obj = self.institute(case_obj["owner"])
+        link = f"/{institute_obj['_id']}/{case_obj['display_name']}"
```

Comment:
`url_for()` should be better here?

Reply:
Mmm I wouldn't use url_for for a path that gets saved to the database. What if you change the name of the domain of our Scout website? I think this was the general idea also for the links that get created in the other types of events. Example:
<img width="322" alt="image" src="https://user-images.githubusercontent.com/28093618/174723123-fb685663-1414-496b-a11f-e816a4e86d70.png">

I'm just keeping the links in the same way

Reply:
> Mmm I wouldn't use url_for for a path that gets saved to the database. What if you change the name of the domain of our Scout website? I think this was the general idea also for the links that get created in the other types of events. Example:
> 
> <img alt="image" width="322" src="https://user-images.githubusercontent.com/28093618/174723123-fb685663-1414-496b-a11f-e816a4e86d70.png">
> 
> I'm just keeping the links in the same way

The risk that the app url routing changes seems more obvious? There is no need to use an `_external=True ` full path with `url_for()`, you could just make a relative? That should be the default for a link that gets rendered for "internal" use. I'm sure there are some exceptions (which probably should change, but I think you will find that the majority use url_for, e.g. https://github.com/Clinical-Genomics/scout/blob/1aac958da9178fd3387b2ed9e2d62bf3d813ca67/scout/server/blueprints/cases/views.py#L631.

Reply:
😊 But perhaps I misunderstood you example? I thought the ones above had been generated by `url_for`? https://github.com/Clinical-Genomics/scout/blob/1aac958da9178fd3387b2ed9e2d62bf3d813ca67/scout/server/templates/utils.html#L71

Reply:
...and here for the status change:
https://github.com/Clinical-Genomics/scout/blob/1aac958da9178fd3387b2ed9e2d62bf3d813ca67/scout/server/blueprints/cases/views.py#L514